sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3014,6 +3014,15 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("reset")
     }
   }
+
+  test("SPARK-27233 match schema when DF is converted to table") {
+    withTable("testSchema") {
+      val df = Seq(Some(Seq(1L, 2L, 3L)), None).toDF("seq")
+      df.write.format("parquet").saveAsTable("testSchema")
+      val res = sqlContext.table("testSchema")
```

Comment:
Not a big deal but let's use spark sessions(not SQL context) next time in the test.